cypress/integration/InstancesTest.ts

```diff
@@ -79,6 +69,95 @@ describe("Instances Test", () => {
         .ouiaId(newInstanceName, "PF4/TableRow")
         .should("not.exist");
     });
+
+    describe("Instance List pagination test", () => {
+      it("Select other page", () => {
+        cy.get("[aria-label='Go to first page']").should("be.disabled");
+        cy.get("[aria-label='Go to previous page']").should("be.disabled");
+        cy.get("[aria-label='Go to last page']").should("not.be.disabled");
+        cy.get("[aria-label='Go to next page']").should("not.be.disabled");
+
+        // Go to next page
+        cy.get("[aria-label='Go to next page']").first().click();
+        cy.get("[aria-label='Go to first page']").should("not.be.disabled");
+        cy.get("[aria-label='Go to previous page']").should("not.be.disabled");
+
+        // as currently count of instances are covered in 2 pages
+        cy.get("[aria-label='Go to last page']").should("be.disabled");
+        cy.get("[aria-label='Go to next page']").should("be.disabled");
+
+        // Go to previous page
+        cy.get("[aria-label='Go to previous page']").first().click();
+      });
+
+      it("Change page size", () => {
+        cy.ouiaType("PF4/DropdownToggle").should("be.visible");
+        cy.ouiaId(
+          "OUIA-Generated-DropdownToggle-1",
+          "PF4/DropdownToggle"
+        ).click();
+        cy.get(".pf-c-options-menu__menu")
+          .find("li")
+          .then(($cells) => {
+            expect($cells).have.length(4);
+            expect($cells.eq(0)).have.text("10 per page");
+            expect($cells.eq(1)).have.text("20 per page");
+            expect($cells.eq(2)).have.text("50 per page");
+            expect($cells.eq(3)).have.text("100 per page");
+          });
+
+        // page size is changed
+        cy.contains("20 per page").click();
+        cy.get(".pf-c-pagination__total-items >b:nth-of-type(1)").then(
+          (count) => {
+            let size = count.text();
+            let perPageSize = parseInt(size.slice(-2));
+            cy.get(".pf-c-pagination__total-items >b:nth-of-type(2)").then(
+              (count) => {
+                let pageSize = parseInt(count.text());
+                cy.wrap(perPageSize).should("be.lte", pageSize);
```

Comment:
Could we please ad assertion that `pageSize` is equal to 20?

Reply:
@jomarko 
UI shows the listing of instances as

1. 20 per page
![image](https://user-images.githubusercontent.com/99238201/183880881-8272e590-ab0d-4988-86ae-b5e0c0f973f3.png)

2. 10 per page


Thus, pageSize should be equal to 'total no of instances' not pageSize

Reply:
+1